### DISCUSSION TITLE
Displaying match result only

### DISCUSSION THREAD

**Discussion by alicemara** [jertel/elastalert2#1251] [Q&A] [answered]
I am trying to figure out a good way to only show values of a field that matches a filter in a jira alert.
I'm looking for all results on a wildcard filter that's formatted like "email: *domain.com"

So if I had three matches aggregated over a day, I would hopefully end up with something like this in the body:

example1@domain.com
example2@domain.com
example3@domain.com

The closest I've gotten is:
```
alert_text: "{0}"
alert_text_args:
- email
```

The above txt will give me the match, but then follows it with the json of the the entire doc the match is part of. 
I have tried with alert_text_type: alert_text_only as well, but that only shows the first result.

Any suggestions are highly appreciated!

**Comment by jertel** [comment, marked as answer]
Have you tried using the [summary_table_fields](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#summary-table-fields) setting? There's some additional documentation on this under the [Aggregation configuration setting](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#aggregation).

**Reply by alicemara** [reply]
I had tried the summary_table_fields, but had missed the aggregation_key option, which looks like it's exactly what I am looking for. Thank you so much!
